nano/node/bootstrap/bootstrap_server.cpp

```diff
@@ -239,10 +239,24 @@ void nano::bootstrap_server::receive_header_action (boost::system::error_code co
 				}
 				case nano::message_type::telemetry_req:
 				{
-					node->stats.inc (nano::stat::type::bootstrap, nano::stat::detail::telemetry_req, nano::stat::dir::in);
 					if (is_realtime_connection ())
 					{
-						add_request (std::make_unique<nano::telemetry_req> (header));
+						auto find_channel (node->network.tcp_channels.find_channel (remote_endpoint));
```

Comment:
I think better make this check in another thread i.e. in request_response_visitor

Reply:
Or `last_telemetry_req `can be variable in boostrap_server instead

Reply:
Moved to bootstrap_server, thanks